specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkSecurityPerimeterGet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Get Network Security Perimeter",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkSecurityPerimeterName": "nsp1"
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
>"body" [](http://example.com/codeflow?start=6&length=6)

Your get call also returns an "etag" property right? might be a good idea to include it here as well to prevent confusions (mainly because your other examples do include an etag property)

Reply:
Added the same.